pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -186,7 +186,7 @@ func (d *DependenciesDistributor) reconcileResourceTemplate(key util.QueueKey) e
 
 		matched := matchesWithBindingDependencies(resourceTemplateKey, binding)
 		if !matched {
-			klog.V(4).Infof("No need to sync binding(%s/%s)", binding.Namespace, binding.Name)
+			klog.V(5).Infof("No need to sync binding(%s/%s)", binding.Namespace, binding.Name)
```

Comment:
The logic here is more like selecting a small number of items from a large set, logging for items that do not match isn't really helpful, right?

In other words, shall we remove this log?

Reply:
Sounds good. We already have the matching logs, so the rest is negligible unmatched resources.

Reply:
I agree to remove this log